kolibri/plugins/learn/assets/src/views/ContentCard/index.vue

```diff
@@ -9,9 +9,21 @@
     ]"
     :style="{ backgroundColor: $themeTokens.surface }"
   >
+    <div class="header-bar">
+      <KLabeledIcon
+        :icon="kind === 'topic' ? 'topic' : `${kindToLearningActivity}Solid`"
+        :label="coreString(kindToLearningActivity)"
+        class="k-labeled-icon"
+      />
+      <img
+        :src="channelThumbnail"
+        :alt="$tr('logo', { channelTitle: channelTitle })"
```

Comment:
It's a bit difficult for me to judge from the code, but is there a visible string with channel title adjacent to this thumbnail image? 

If it is, than the `alt` attribute would essentially just make the screen reader read the same information twice, and that is redundant. In that case, I suggest an empty `alt` attribute to make the image decorative, which should take it out of the accessibility tree and not being announced by the screen readers.

Reply:
hi @radinamatic - happy to "translate" the code into something more understandable. This logo is the channel logo that is visible here in the new card design, where there isn't any other reference to the channel title present: 
<img width="539" alt="Screen Shot 2021-09-14 at 10 17 50 AM" src="https://user-images.githubusercontent.com/17235236/133274870-55390e6f-1a49-4269-b32c-4c6dbc5f2429.png">
In this case, based on the situations you've described, it seems like it might be good to still have the alt text? Is there different alt text I should include that's more helpful?

The other situations you find below are similar cases, just where the card is a different layout